custom/icds_reports/tasks.py

```diff
@@ -1143,3 +1178,25 @@ def _bust_awc_cache():
     for key in reach_keys:
         cache.delete(key)
     create_datadog_event('redis: delete dashboard keys', 'finish')
+
+
+# TODO create new queue
+@task
+def run_citus_experiment(fn, args, kwargs):
+    experiment_context = {
+        "function_name": fn.__name__,
+        'args': args,
+        'kwargs': kwargs,
+    }
+    experiment = DashboardQueryExperiment(name="Dashboard Query Experiment", context=experiment_context)
+    with experiment.control() as c:
+        c.record(fn(*args, **kwargs))
+
+    with experiment.candidate() as c:
+        databases = settings.DATABASES
+        databases['icds-ucr'] = deepcopy(databases['citus'])
```

Comment:
I think you did not want to overwrite `settings.DATABASES['icds-ucr']` with a copy of the `citus` database (`override_settings` is not even necessary if you do that). You probably wanted this instead
```py
databases = settings.DATABASES.copy()
databases['icds-ucr'] = databases['citus']
```
I think this is a good example of why it's dangerous to be mutating `settings` values in production code.